File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioEntActionMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdAction;

import java.util.Map;

@SuppressWarnings("unused")
public class StudioEntActionMap extends StudioBase
{
  public MetaIdAction[] keys;

  public Map<MetaIdAction, StudioEntAction> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntActionMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdAction

interface StudioEntActionMap : StudioBase {
    val keys: List<MetaIdAction>
    val map: Map<MetaIdAction, StudioEntAction>
}
